test/test_helper.rb

```diff
@@ -43,6 +43,8 @@ def sleep_until(*args, &block)
     sleep_until?(*args, &block) || raise("sleep_until timeout reached")
   end
 
+  ruby2_keywords(:sleep_until) if respond_to?(:ruby2_keywords, true)
+
   def sleep_until?(timeout: SLEEP_UNTIL_TIMEOUT)
```

Comment:
This is only used within this file. Rather than using `ruby2_keywords`, I say we just move to using a positional param and continuing to use `*args` to forward. I don't even think `SLEEP_UNTIL_TIMEOUT` is defined anywhere, so this is ripe for a refactor. I wonder why it's not using the Timeout module... but I digress.

Reply:
OK, I will update it to remove dependency on ruby2_keywords